### PR TITLE
Pypy hasattr

### PR DESCRIPTION
**Pull request by ghuls**
Thank you for contributing to pyo3!

Please consider adding the following to your pull request:
 - an entry in CHANGELOG.md
 - docs to all new functions and / or detail in the guide
 - tests for all new or changed functions

Be aware the CI pipeline will check your pull request for the following:
 - Rust tests (Just `cargo test` or `make test` if you need to test examples)
 - Rust lints (`make clippy`)
 - Rust formatting (`cargo fmt`)
 - Python formatting (`black . --check`. You can install black with `pip install black`)
 - Compatibility with all supported Python versions for all examples. This uses `tox`; you can do run it using `make test_py`.

You can run a similar set of checks as the CI pipeline using `make test`.

### CODE REVIEW

**Comment by birkenfeld** [comment]
The first commit here looks unrelated?

**Comment by ghuls** [comment]
> The first commit here looks unrelated?

Is removed now. was a leftover from testing. rust-numpy does not support the latest pyo3 version (or at least I couldn't get it to work).

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add `Py::setattr` method. [#2009](https://github.com/PyO3/pyo3/pull/2009)
 - Add `PyCapsule`, exposing the [Capsule API](https://docs.python.org/3/c-api/capsule.html#capsules). [#1980](https://github.com/PyO3/pyo3/pull/1980)
+- Map `PyPyObject_HasAttr` to `PyObject_HasAttr` so `hasattr` works with PyPy. [#2025](https://github.com/PyO3/pyo3/pull/2025)
```
Comment (davidhewitt) [inline CHANGELOG.md:24]: I'm going to force-push to move this to the `## Fixed` section.

**Review by davidhewitt — APPROVED** [review]
Thanks!

**Pull request merged**